web3-utils/contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.22 <0.9.0;
+
+contract Migrations {
```

Comment:
Why are we using `Migrations.sol`?

I was curious about that, as we could probable use a much simpler contract which already  includes storage and serves for other purposes.

The one @ed255 and I have been using is this:
```solidity
// SPDX-License-Identifier: GPL-3.0

pragma solidity >=0.7.0 <0.9.0;

/**
 * @title Greeter
 * @dev Store & retrieve value in a variable
 */
contract Greeter {

    uint256 number;
    
    constructor(uint256 num) {
        number = num;
    }

    function retrieve() public view returns (uint256){
        return number;
    }
    
    function retrieve_failing() public view returns (uint256){
        require(false);
        return number;
    }
   
   function set_value(uint256 num) public{
        number = num;
    }
    
    function set_value_failing(uint256 num) public{
        number = num;
        require(false);
    }
}
```

Reply:
I replaced with this contract!